dmutils/email/dm_mailchimp.py

```diff
@@ -161,3 +163,35 @@ def get_email_addresses_from_list(self, list_id, pagination_size=100):
             offset += pagination_size
 
             yield from [member['email_address'] for member in member_data['members']]
+
+    def get_lists_for_email(self, email_address: str) -> Sequence[Mapping]:
```

Comment:
I like typing, but I thought when I tried this we agreed not to do it piecemeal?

Reply:
I don't remember how much I was on board with that - I think it's *specifically* useful here because it's an API method, and helps explain the interface.

Reply:
(I wonder if newer versions of `mock` are considering type-checking autospecced functions that have typing...?)

Reply:
Are you trying to tempt me to do the whole class?